src/gnn_reco/data/i3extractor.py

```diff
@@ -74,6 +75,13 @@ def __call__(self, frame) -> dict:
             'width' : [],
             'pmt_area': [],
             'rde': [],
+            'string': [],
+            'pmt_number': [],
+            'dom_number': [],
+            'pmt_dir_x': [],
+            'pmt_dir_y': [],
+            'pmt_dir_z': [],
+            'dom_type': [],
```

Comment:
@RasmusOrsoe here, and below (L97-103) an Upgrade-specific change seems to have snuck in. Is this intentional? Because I suspect this might give errors when running the existing example code/using the default GCD files. Would it be okay to remove these changes for this PR, and include them as part of a separate, Upgrade-specific PR?